### PR TITLE
Add Weekly Email Digest (first pass render)

### PR DESCRIPTION
**Pull request by copycut**

### CODE REVIEW

File: src/helpers/getPercentPropotionnal.js [hunk 1]
```diff
@@ -0,0 +1,37 @@
+module.exports.getPercentPropotionnal = function (items, item, key) {
```
Comment (torsjonas) [inline src/helpers/getPercentPropotionnal.js]: Spelling is proportional

File: src/helpers/abs.js [hunk 2]
```diff
@@ -0,0 +1,3 @@
+module.exports.abs = function (number) {
```
Comment (torsjonas) [inline src/helpers/abs.js]: Maybe a bit redundant to create a one line wrapper around the Math.abs function? Or maybe there is a good reason for this?

Comment (copycut) [inline src/helpers/abs.js]: The purpose is to use Math.abs with Handlebars custom helper:
```hbs
{{abs percent}}
```

Comment (torsjonas) [inline src/helpers/abs.js]: Ah ok!

**Pull request merged**